test/vtl.test.ts

```diff
@@ -207,7 +208,8 @@ test("if statement", () => {
       }
     }),
     payload,
-    `#if($context.arguments.list.length > 0)
+    `#set($v1 = $context.arguments.list.length > 0)
+#if($v1)
```

Comment:
these changes are a little annoying, we don't need to set a variable if the parent node is an `IfStmt`, but I'll leave it as is for consistency unless someone feels strongly about it